api.py

```diff
@@ -0,0 +1,76 @@
+import os
+import logging
+import requests
+import flask
+import datetime
+
+
+from flask import Flask, request, flash, redirect, render_template, url_for, jsonify
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
```

Comment:
the config is loaded from a file that is defined using the `FLASK_SETTINGS` env variable (see: https://github.com/GreatLakesEnergy/sesh-api-helper/pull/1/files#diff-33c3175096a7d335a8cec920d164ef63R19 )

afaik the __name__ is more the internal name. If we are not in a module it is safe to use `__name__`